homework_02/Martin_Georgiev_13/views.py

```diff
@@ -5,19 +5,67 @@
 
 def trains(request):
 
-    cur_time = datetime.datetime.now(pytz.timezone('America/New_York'))
-    response = requests.get("https://api-v3.mbta.com/predictions?page%5Boffset%5D=0&page%5Blimit%5D=10&sort=-departure_time&filter%5Bstop%5D=place-north").json()
+    params = {
+        "sort" : "-arrival_time",
+        "filter[direction_id]" : "0",
+        "page[offset]" : "0",
+        "include" : "trip,schedule",
+        "page[limit]" : "100",
+        "filter[route_type]" : "2",
+        "filter[stop]" : "place-north",
+        "filter[direction_id]" : "0",
+    }
+
+    current_time = datetime.datetime.now(pytz.timezone("America/New_York"))
+    response = requests.get("https://api-v3.mbta.com/predictions", params = params).json()
+
+    destination_times = []
+    trip_ids = []
+    directions = []
     statuses = []
-    for status in response['data']:
-        statuses.append(status['attributes']['status'])
+    train_ids = []
```

Comment:
Cool, you have splitted the different fields of the information in different lists!